package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "data-transparency-ui",
-  "version": "6.2.7",
+  "version": "6.2.8",
   "description": "Library of UI components powering Data Transparency websites",
-  "main": "dist/index.js",
+  "main": "index.js",
```

Comment:
do you need to undo this? i think this was an npm link change

Reply:
yes, i do, thanks